client/components/App.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import styled from 'styled-components';
+import TimeListContainer from '../redux/containers/TimeListContainer';
+import AddressContainer from '../redux/containers/AddressContainer';
+import PhoneNumberContainer from '../redux/containers/PhoneNumberContainer';
+import WebsiteContainer from '../redux/containers/WebsiteContainer';
+import GetDirectionsContainer from '../redux/containers/GetDirectionsContainer';
+import MapContainer from '../redux/containers/MapContainer';
+import ModalContainer from '../redux/containers/ModalContainer';
+
+const Module = styled.div`
+  padding: 32px;
+  margin: 32px;
+  margin-right: 150px;
+  min-width: 22vw;
+  max-width: 330px;
+  max-height: 80vh;
+  background-color: white;
+  float: right;
+  overflow: scroll;
+
+  @font-face {
+    font-family: "Calibre-Regular";
+    src: url("../../public/fonts/CalibreWeb-Regular.woff2") format("woff2");
+  }
+`;
+
+const Spacer = styled.div`
+  width: 100%;
+  height: 25px;
+`;
+
```

Comment:
Is there a better way to add blank space in between your different subcomponents than with this 'Spacer' element?

I know it's way easier to insert random < div >'s and < br >'s (my own specialty!), and I think that's fine for early development...but replacing this 'Spacer' element with more-specific CSS modules (or styled React components, in your case) would make your /App.jsx file not only a lot D.R.Y.-er ("Don't Repeat Yourself"), but also a lot cleaner and easier to maintain.